Wenab/Common/LittleByte.Common/AspNet/AutoMapper/NullObjectMapper.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Linq.Expressions;
+using AutoMapper;
+
+namespace LittleByte.Common.AspNet.AutoMapper;
+
+public class NullObjectMapper : IMapper
+{
+    public TDestination Map<TDestination>(object source)
+    {
+        return default!;
```

Comment:
Good question but no 😁 `AutoMapper` is a library that converts objects and is generally used to translate data between layers (eg: DAO from the data layer to a domain model). This is just a null implementation for testing purposes. At least I think since I just copied it from my common repo 😅